x/oracle/keeper/grpc_query.go

```diff
@@ -249,3 +249,10 @@ func (q querier) AggregateVotes(
 		AggregateVotes: votes,
 	}, nil
 }
+
+// Medians currently performs a no-op.
+func (q querier) Medians(
+	_ context.Context, _ *types.QueryMedians,
+) (*types.QueryMediansResponse, error) {
+	return &types.QueryMediansResponse{}, nil
```

Comment:
Creating an `ErrNotImplemented` would be good here rather than returning empty response but `nil` error